### PR TITLE
PLZ for liechtentein beginning 948 or 949 should be filtered

### PR DESCRIPTION
**Pull request by hamidsoleymani**
Please dont merge!

### CODE REVIEW

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/locality/LocalitySearchDto.java [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
     private final boolean distinctByLocalityCity;
+    private final String countryCode;
 
-    public LocalitySearchDto(String query, int size, boolean distinctByLocalityCity) {
-        this.query = query;
+    public LocalitySearchDto(String prefix, int size, boolean distinctByLocalityCity, String countryCode) {
+        this.query = prefix;
         this.size = size;
         this.distinctByLocalityCity = distinctByLocalityCity;
+        this.countryCode = StringUtils.isBlank(countryCode) ? null : countryCode.toUpperCase();
```
Comment (birom) [inline reference-app/src/main/java/ch/admin/seco/service/reference/service/locality/LocalitySearchDto.java]: There is a ``StringUtils.toUpperCase`` method.

Comment (hamidsoleymani) [reply reference-app/src/main/java/ch/admin/seco/service/reference/service/locality/LocalitySearchDto.java]: ok

File: reference-app/src/main/java/ch/admin/seco/service/reference/service/locality/DefaultLocalityAutocompleteConverter.java [hunk 2]
```diff
@@ -45,6 +43,7 @@ protected String getMapKey(LocalitySuggestionDto dto) {
         return getSuggestOptionsStream(searchResponse, "cities", "zipCodes")
             .map(this::toLocalitySuggestionDto)
             .filter(Objects::nonNull)
+            .sorted((o1, o2) -> o2.getCantonCode().equals(LIECHTENSTEIN) ? -1 : 0) // put all Liechtenstein cantons to the end of the list and remain the original sort order
```
Comment (cjpsycos) [inline reference-app/src/main/java/ch/admin/seco/service/reference/service/locality/DefaultLocalityAutocompleteConverter.java]: why are they not filtered out?

Comment (hamidsoleymani) [reply reference-app/src/main/java/ch/admin/seco/service/reference/service/locality/DefaultLocalityAutocompleteConverter.java]: Filtering is done in frontend! See here:
https://github.com/alv-ch/alv-portal/pull/751/files

**Pull request merged**